docs/source/development.rst

```diff
@@ -125,6 +131,32 @@ not have binaries built for the M1. Need to install using an x86_64 version of P
 are issues installing `biofomats2raw` as there is no OpenCV package for arm64 chip. There currently
 is no fix or workaround for this issue.
 
+Docker
+======
+
+You can also choose to use Docker for local development and testing.
+
+In order to build the docker image, use `--platform linux/amd64` option.
+Explanation can be found `here <https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::5f55363b-bb53-4e5b-9564-8bed5289fdd5?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A15%2C%5C%22sectionId%5C%22%3A17%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3A869be6677ee54848bc13f2066d847cc0%40thread.skype%22%7D&tenantId=14b77578-9773-42d5-8507-251ca2dc2b06>`_
+
+The basic usage for testing would look like such:
+
+.. code-block:: sh
+
```

Comment:
can we specify which Dockerfile we're going to use here?
eg `repo_root/Dockerfile`